### ISSUE TITLE
Broken bash completion

### ISSUE THREAD

**Issue by candlerb** [streamnative/pulsarctl#138]
* Type "./pulsarctl topics de"
* Hit Tab

Result:

```
$ ./pulsarctl topics de-su: aliashash[""]: bad array subscript
```

**Comment by wolfstudy** [comment]
@candlerb see https://github.com/streamnative/pulsarctl/blob/master/docs/en/enable_completion.md

**Comment by bublikhpt** [comment]
I have the same problem on linux systems (centos 7, centos 8, debian testing) any of them doesn't work.
```
$ pulsarctl bash: aliashash[""]: bad array subscript
$ pulsarctl topbash: aliashash[""]: bad array subscript
$ pulsarctl topics bash: aliashash[""]: bad array subscript
```

**Comment by lyra95** [comment]
A simple repro to this issue:

```Dockerfile
FROM ubuntu:24.04

RUN apt-get update && apt-get install -y curl bash-completion

RUN bash -c "$(curl -fsSL https://raw.githubusercontent.com/streamnative/pulsarctl/master/install.sh)"

RUN echo 'export PATH=${PATH}:${HOME}/.pulsarctl/plugins' >> ~/.bashrc \
        && echo "source /etc/profile.d/bash_completion.sh" >> ~/.bashrc \
        && echo ". <(pulsarctl completion bash)" >> ~/.bashrc
FROM ubuntu:24.04

RUN apt-get update && apt-get install -y curl bash-completion

RUN bash -c "$(curl -fsSL https://raw.githubusercontent.com/streamnative/pulsarctl/master/install.sh)"

RUN echo 'export PATH=${PATH}:${HOME}/.pulsarctl/plugins' >> ~/.bashrc \
        && echo "source /etc/profile.d/bash_completion.sh" >> ~/.bashrc \
        && echo ". <(pulsarctl completion bash)" >> ~/.bashrc
```

```terminal
❯ docker run --rm -it pulsarctl:test bash
root@18000a6fb81c:/# echo $BASH_VERSION
5.2.21(1)-release
root@18000a6fb81c:/# pulsarctl combash: aliashash[""]: bad array subscript
root@18000a6fb81c:/#
```

**Comment by lyra95** [comment]
@wolfstudy created a fix pr(#1683), could you check it out?
